src/cluster_info.rs

```diff
@@ -274,6 +296,24 @@ impl ClusterInfo {
             .collect()
     }
 
+    pub fn sorted_tvu_peers(&self, bank: &Arc<Bank>) -> Vec<NodeInfo> {
+        let peers = self.tvu_peers();
+        let mut peers_with_stakes: Vec<(&NodeInfo, u64)> =
+            peers.iter().map(|c| (c, bank.get_stake(&c.id))).collect();
+        peers_with_stakes.sort_unstable_by(|l, r| {
+            let cmp = r.1.cmp(&l.1);
```

Comment:
This closure makes my brain hurt. If you added a custom `Ord` instance for `NodeInfo`, would this reduce to `peer_with_stakes.sort_unstable()`?

Reply:
Unless I'm missing something I don't think that's enough. I need to sort this vec of tuples. So I'd need to define a Ord for (&NodeInfo, u64) probably and that seems more tedious. Certainly more so than this? 

I can be more explicit about what is being sorted, something like this maybe and throw in a comment?

```
peers_with_stakes.sort_unstable_by(|(lContactInfo, lStake), (rContactInfo, rStake)| {
            let order = rStake.cmp(&lStake.1);
            if order == cmpOrdering::Equal {
                // fallback to comparing pubkeys if stakes are equal
                rContactInfo.id.cmp(&lContactInfo.id)
            } else {
                order
            }
        });
```

Reply:
The Ord instance for a tuple is generic. It's something like:

```rust
if cmp(l.0, r.0) == Ordering::Equal { cmp(l.1, r.1) } else {cmp(l.0, r.0)}
```

So all you should need is the `Ord` instance for `NodeInfo`.

Reply:
Oh, did not realize that. Alright I'll see if i can do this.